### ISSUE TITLE
@rjsf/material-ui 2.0.0-alpha.4 is not built properly for cjs

### ISSUE THREAD

**Issue by heath-freenome** [rjsf-team/react-jsonschema-form#1677]
### Prerequisites

- [x] I have read the [documentation](https://react-jsonschema-form.readthedocs.io/);
- [x] In the case of a bug report, I understand that providing a [SSCCE](http://sscce.org/) example is tremendously useful to the maintainers.
- [ ] Ideally, I'm providing a [sample JSFiddle](https://jsfiddle.net/n1k0/f2y3fq7L/6/) or a [shared playground link](https://rjsf-team.github.io/react-jsonschema-form/) demonstrating the issue.

### Description

I'm trying to write a test that uses the `Form` from `@rjsf/material-ui` using jest.  Jest fails for me with the following error:
```
<project_root>/node_modules/@rjsf/core/lib/utils.js:1
    ({"Object.<anonymous>":function(module,exports,require,__dirname,__filename,global,jest){import _Symbol$toPrimitive from "@babel/runtime-corejs2/core-js/symbol/to-primitive";
                                                                                                    ^^^^^^^^^^^^^^^^^^^

    SyntaxError: Unexpected identifier

      at Runtime.createScriptFromCode (node_modules/jest-runtime/build/index.js:1059:14)
      at Object.<anonymous> (node_modules/@rjsf/material-ui/dist/material-ui.cjs.development.js:9:13)
```

After looking at `node_modules/@rjsf/material-ui/dist/material-ui.cjs.development.js` I can see that the generated code is directly referencing the raw ES6 code rather than the `@rjsf/core/dist/cjs/utils`.   See the `<<<<<<<<<<<<<` below in the extract from that file.

```
'use strict';

Object.defineProperty(exports, '__esModule', { value: true });

function _interopDefault (ex) { return (ex && (typeof ex === 'object') && 'default' in ex) ? ex['default'] : ex; }

var core = require('@rjsf/core');
var React = _interopDefault(require('react'));
var utils = require('@rjsf/core/lib/utils');   <<<<<<<<<<<<<<<
var Box = _interopDefault(require('@material-ui/core/Box'));
var Grid = _interopDefault(require('@material-ui/core/Grid'));
var Paper = _interopDefault(require('@material-ui/core/Paper'));
...
```

If I manually fix this file locally to change `@rjsf/core/lib/utils` to `@rjsf/core/dist/cjs/utils` then my tests pass.  Although given how this one require was incorrect and how the remainder of them also seem to load up the raw Typescript code, I have to wonder if the remainder of the `@rjsf/material-ui` cjs distribution also needs to be properly transpiled down to CommonJS like is done for the `@rjsf/core` distribution.

### Steps to Reproduce

1. [First Step]
2. [Second Step]
3. [and so on...]

#### Expected behavior

The material-ui cjs distribution is all CommonJS

#### Actual behavior

The material-ui cjs distribution is a mix of CommonJS and raw Typescript code

### Version

@rsjf/material-ui 2.0.0-alpha.4

**Comment by heath-freenome** [comment]
I'm fairly certain that the Typescript code needs to be compile down to CommonJS because when I try mounting my component in my jest test node blows up.  My other tests were simply shallow mounting my component, which does not render the inner `Form`

**Comment by heath-freenome** [comment]
@epicfaace I hope this gets fixed before you release the `2.0.0` version because it's pretty broken from the testing perspective.  Whatever is done to make the `@rjsf/core` release compile properly, please do it for `@rjsf/material-ui`

**Comment by heath-freenome** [comment]
@epicfaace [issue](https://github.com/rjsf-team/react-jsonschema-form/issues/1655) is completed, it may solve the initial part of this since the `utils` library is imported directly rather than through the `@rsjf/core` since the core `index.js` does not export it.  It seems like an easy thing to do
